### PR TITLE
Add drush deploy step

### PR DESCRIPTION
**Pull request by chrfritsch**
### 💬 Describe the pull request/code changes
Add the drush deploy command

### 🔢 To Review
_Steps to review the PR:_
1. 
2. 

### 📝 Checklist
- [ ] Update the "Unreleased" section of the `CHANGELOG.md` with [chan](https://github.com/geut/chan/tree/main/packages/chan).  _(if applicable)_
- [ ] The PR is covered by some tests. _(if applicable)_

[comment]: <> (Add the Jira ticket if needed/applicable)
[comment]: <> (Add steps to review the PR if needed/applicable)
[comment]: <> (Add others conditions to allow the PR to be merged if needed/applicable)

### CODE REVIEW

**Review by WengerK — APPROVED** [review]

File: lib/capdrupal.rb [hunk 1]
```diff
@@ -191,6 +191,15 @@
     end
   end
 
+  desc 'Run several commands after performing a code deployment'
```
Comment (gido) [inline lib/capdrupal.rb:194]:
```suggestion
  desc 'Run several commands after performing a code deployment (drush deploy)'
```

**Review by gido — APPROVED** [review]

**Comment by WengerK** [comment]
A huge thank you for your PR @chrfritsch (and your incredible patience awaiting a response — it's truly appreciated)

Regarding the proposed code, I wanted to check in with you since it's been two years since you submitted it.
Do you still feel that integrating `drush:deploy` within Capistrano is relevant? Does it align with Drupal current needs and practices?

Thanks for your help

**Comment by chrfritsch** [comment]
Hi,
definetly. It shouldn't used anything else. drush:deploy does everything you need (database update, config import, cache clear)

**Comment by gido** [comment]
Thanks!

**Pull request merged**